gbenchmarks/benchmark/.clang-format

```diff
@@ -0,0 +1,5 @@
+---
+Language:        Cpp
+BasedOnStyle:  Google
```

Comment:
Should there be a download script for this instead ? That is, use `FindGTest` first. If not found on system, download it.
I simply copied the folder, since a similar thing was done for `gtest`.

Reply:
Please let me handle these changes. I actually want to remove gtest and use external versions of both gtest and gbenchmark, to avoid that the repository becomes too big. Downloading is totally out of question, the build should not depend on a network connection.

Reply:
Yes. Sounds logical.